pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -810,7 +810,7 @@ class DelayedSaturatedMMM(
             yearly_seasonality=2,
         )
 
-    Observe we can choose the distribution and all parameters of the model.
+    We can choose the distribution and all parameters and likelihood of the model through the `model_config` parameter.
```

Comment:
"As you can see, we can configure all prior and likelihood distributions via the `model_config`."?

Reply:
👍 Added!